app/views/docs/database.phtml

```diff
@@ -1,22 +1,35 @@
 <p>
     The Database Service allows you to store your application and users' data and fetch it using different supported queries.
```

Comment:
The `Database` service should now be `Databases` everywhere

Reply:
fixed